docs/source/api/pipelines/stable_diffusion.mdx

```diff
@@ -73,16 +73,18 @@ If you want to use all possible use cases in a single `DiffusionPipeline` you ca
 
 ## StableDiffusionPipeline
 [[autodoc]] StableDiffusionPipeline
+	- all
 	- __call__
 	- enable_attention_slicing
 	- disable_attention_slicing
-	- enable_vae_slicing
-	- disable_vae_slicing
 	- enable_xformers_memory_efficient_attention
 	- disable_xformers_memory_efficient_attention
 
+
+
 ## StableDiffusionImg2ImgPipeline
 [[autodoc]] StableDiffusionImg2ImgPipeline
+	- all
 	- __call__
 	- enable_attention_slicing
```

Comment:
exceptionally I'm duplicating all those `enable_attention_slicing` from `DiffusionPipeline` here since this doc page is viewed a lot.